packages/react/src/vm-extension/node-vm-ext-list.tsx

```diff
@@ -21,38 +21,58 @@ export const NodeVMExtList = (props: NodeVmExtensionListProps) => {
         return inject(BatchDependencyName.NodeService);
     }, []);
 
+    const accountService: AccountService = React.useMemo(() => {
```

Comment:
This is totally fine, but FYI there's not much reason to use `useMemo()` here as `inject()` is a very cheap function and should return a stable value.

Reply:
(Feel free to resolve this with no change - just wanted to point this out)

Reply:
Maybe we can make it a custom hook like `useAccountService` and use `useMemo` inside.

Reply:
Eh, this isn't ever going to be a performance problem - it's just calling a function and reading a property on an object. I'd say let's just use `inject` directly rather than wrapping it. Not all function calls inside a React component need to be hooks.